README.md

```diff
@@ -43,18 +43,18 @@ JabRef supports you in every step of your research work.
 - Attach related documents: 20 different kinds of documents supported out of the box, completely customizable and extendable
 - Automatically rename and move associated documents according to customizable rules
 - Keep track of what you read: ranking, priority, printed, quality-assured 
-	
+    
 #### Cite
 
 - Native [BibTeX] and [Biblatex] support
 - Cite-as-you-write functionality for external applications such as Emacs, Kile, LyX, Texmaker, TeXstudio, Vim and WinEdt.
 - Format references in one of the many thousand built-in citation styles or create your own style
 - Support for Word and LibreOffice/OpenOffice for inserting and formatting citations
-	
+    
 #### Share
 
 - Many built-in export options or create your own export format
-- Library is saved as a simple text file and thus it is easy to share with others via Dropbox and is version-control friendly
+- The library is saved as a simple text file and thus it is easy to share with others via Dropbox and is version-control friendly
```

Comment:
I would write "The BibTeX/BibLaTeX library"

Reply:
Maybe BibTex? - Many users do not know that we work with native text files. I wanted to express that here.

Reply:
Maybe "bibliographic library" but in my opinion "library" should be clear enough.